test/testdata/infer/generics/disabled/variance_params.rb

```diff
@@ -0,0 +1,50 @@
+# typed: true
+
+class Animal; end
+class Cat < Animal; end
+
+class Other; end
+
+class Makes
+  extend T::Sig
+  extend T::Helpers
+  extend T::Generic
+
+  Ty = type_member(:out) # error: Classes can only have invariant type members
```

Comment:
could you make these into modules?

Reply:
Done. Thanks for the suggestion, I've re-enabled these tests as this passes in LSP now.